API/application/iotjs.py

```diff
@@ -129,35 +128,34 @@ def build(self):
         '''
         self.update_repository()
 
-        # Note: We should build IoT.js for Raspberry Pi 2, since the
-        # binary size information (showed on the test-result webpages)
-        # is based on this target.
-        minimal_build_flags = [
+        common_build_flags = [
             '--clean',
             '--buildtype=%s' % self.buildtype,
             '--target-arch=arm',
-            '--no-parallel-build',
-            '--target-board=rpi2',
-            '--builddir=%s' % paths.IOTJS_MINIMAL_BUILD_PATH
         ]
 
+        # Note: We should build IoT.js for Raspberry Pi 2, since the
+        # binary size information (showed on the test-result webpages)
+        # is based on this target.
+        minimal_build_flags = list(common_build_flags)
+        minimal_build_flags.append('--target-board=rpi2')
+        minimal_build_flags.append('--builddir=%s' % paths.IOTJS_MINIMAL_BUILD_PATH)
+
+        # Run the buildscript with minimal build flags for binary information.
         utils.execute(paths.IOTJS_PATH, 'tools/build.py', minimal_build_flags)
 
         # The following builds are target specific with memory usage features.
+        build_flags = list(common_build_flags)
```

Comment:
Could we move this line below the `include_modules = ['spi', 'uart']` line? In this case `build_flags` could be in one place.

Reply:
LGTM

Reply:
Fixed.